### PR TITLE
mkcloud: make clouddata hostname variable

### PR DESCRIPTION
**Pull request by bmwiedemann**

### CODE REVIEW

**Comment by toabctl** [comment]
+1

**Comment by rsalevsky** [comment]
+1

**Comment by MaximilianMeister** [comment]
-1 (because no docs)
can you add a line of documentation?

sth like 
```
# clouddata = needs to have subfolder repos, images and so on
```

**Comment by MaximilianMeister** [comment]
maybe I am getting it wrong but shouldn't it be possible to host your own clouddata at some point in time? with all the images/repos/files that we need for mkcloud? then we would not be dependent on clouddata.cloud.suse.de anymore, or on a mirror of it
if we want to go that way, we should add some more detailed comment how to set it up, how it needs to look like and what should be in it. don't know if that makes sense in the long term. any thoughts?

File: scripts/mkcloud [hunk 1]
```diff
@@ -196,6 +196,8 @@ function sshrun()
     cat > $mkcconf <<EOF
         export drbdnode_mac_vol=$drbdnode_mac_vol ;
         export cloud=$virtualcloud ;
+        # hostname of NFS server with repos and images:
```
Comment (MaximilianMeister) [inline scripts/mkcloud]: and it should be in usage, where we have hints about the variables from outside

Comment (bmwiedemann) [inline scripts/mkcloud]: will be documented later when we add the mirror-setup-script

Comment (rsalevsky) [inline scripts/mkcloud]: The latest change make absolute no sense. You added a comment above the line in a section where nowhere else are comments. Why you didn't make it correctly and added it to the usage? It's relatively the same amount of work.

**Pull request merged**